Added quiz functionality with result page

willkodes opened the pull request:
- Created Quiz page (Quiz.js)
- Implemented Question component (Question.js)
- Added Result page to display final results (Result.js)
- Included additional CSS for styling (Question.css, App.css)

willkodes committed:
Added quiz functionality with result page

- Created Quiz page (Quiz.js)
- Implemented Question component (Question.js)
- Added Result page to display final results (Result.js)
- Included additional CSS for styling (Question.css, App.css)

sigehiro commented:
To begin with, there are conflicts. This is because the latest code in the develop branch has not been imported.
I can't even review the file if there are conflicts. To be precise, even if I fix the conflicts here, it may not be what you intended, so I think it would be better if you fix the conflicts.

Please submit a pull request again.

willkodes committed:
TasteQuiz + Top, Header...

Created Quiz page (Quiz.js)
Implemented Question component (Question.js)
Added Result page to display final results (Result.js)
Included additional CSS for styling (Question.css, App.css)
